### ISSUE TITLE
may be Learning Rate should be 0.016?

### ISSUE THREAD

**Issue by syd951186545** [potterhsu/SVHNClassifier-PyTorch#12]
when i traning by using the Learning Rate 0.16,the loss will increase to  nan.
